library/ssl_cli.c

```diff
@@ -829,6 +833,11 @@ static int ssl_write_client_hello( mbedtls_ssl_context *ssl )
         MBEDTLS_SSL_DEBUG_MSG( 3, ( "client hello, add ciphersuite: %2d",
                        ciphersuites[i] ) );
 
+#if defined(MBEDTLS_ECDH_C) || defined(MBEDTLS_ECDSA_C) || \
+    defined(MBEDTLS_KEY_EXCHANGE_ECJPAKE_ENABLED)
```

Comment:
I do not think that Mbed TLS 2.1 has ECJPAKE. Does it? If it does not, are we sure we want to keep these preprocessor conditions with `defined(MBEDTLS_KEY_EXCHANGE_ECJPAKE_ENABLED)`? I dont see a problem with leaving it in other than is a bit unnecessary and slightly inconsistent.

Reply:
good catch!
It was inserted by mistake when I cherry=picked the changes. I thought I removed all references to ECJPAKE